examples/rules/operator/thanos-operator/thanos-operator-alerts.yaml

```diff
@@ -0,0 +1,600 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/component: thanos-operator
+    app.kubernetes.io/name: thanos-operator-rules
+    app.kubernetes.io/part-of: thanos-operator
+    app.kubernetes.io/version: main
+  name: thanos-operator-alerts
+  namespace: monitoring
+spec:
+  groups:
+  - interval: 30s
+    name: thanos-operator.general
+    rules:
+    - alert: ThanosOperatorDown
+      annotations:
+        dashboard: https://demo.perses.dev/projects/perses/dashboards/thanosoperator
+        description: The Thanos Operator has been down for more than 5 minutes. No
+          reconciliation is happening.
+        message: Thanos resources are not being reconciled. Configuration changes
+          will not be applied.
+        runbook: https://github.com/thanos-community/thanos-operator/blob/main/mixin/runbook.md#thanosoperatordown
```

Comment:
that's a 404?

Reply:
Yeah will add a runbook to the repo upstream shortly! But I think most people will end up overriding this with their downstream runbooks.